include/xtensor-io/xio_blosc.hpp

```diff
@@ -107,9 +107,8 @@ namespace xt
         }
     }  // namespace detail
 
-    /**
+    /*
```

Comment:
This star should not be removed, it is required by Doxygen to be interpreted as a documented comment. The same applies everywhere.